src/napari_matplotlib/tests/scatter/test_scatter.py

```diff
@@ -1,6 +1,7 @@
 from copy import deepcopy
 
 import pytest
+from numpy import array
```

Comment:
```suggestion
import numpy as np
```
Since this is the usual way to import numpy